src/ccs/cccs.rs

```diff
@@ -28,47 +28,64 @@ use std::sync::Arc;
 
 use super::util::compute_sum_Mz;
 use super::util::virtual_poly::VirtualPolynomial;
-use super::{CCSShape, CCSWitness};
+use super::CCS;
 
 /// A type that holds the shape of a Committed CCS (CCCS) instance
-#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
-#[serde(bound = "")]
-pub struct CCCSShape<G: Group> {
-  // Sequence of sparse MLE polynomials in s+s' variables M_MLE1, ..., M_MLEt
-  pub(crate) M_MLE: Vec<MultilinearPolynomial<G::Scalar>>,
-
-  pub(crate) ccs: CCSShape<G>,
+#[derive(Clone, Debug, PartialEq, Eq /*Serialize, Deserialize*/)]
+//#[serde(bound(deserialize = "'de: 'a"))]
+pub struct CCCS<G: Group> {
```

Comment:
Calling this just CCCS seems a bit weird to me since it stands for "Committed Customizable Constraint **Systems**". Should this be CCCS Shape or Instance instead?

Reply:
Not sure it is a big deal though. In any case, I can definitely change it. What do the rest think?

Reply:
I actually think is better to have shorter names. Also, we have always called it CCCS indeed in our calls. So that's why I considered a better alternative.

Reply:
I guess point would be to stay consistent with code base, but I don't feel super strongly about it, can see both sides